usmqe/web/tendrl/mainpage/clusters/cluster/pages.py

```diff
@@ -0,0 +1,116 @@
+"""
+Page objects for cluster details navigation
+
+Author: ltrilety
+"""
+
+
+import copy
+
+from webstr.core import WebstrPage
+
+import usmqe.web.tendrl.mainpage.clusters.cluster.models as m_cluster
+from usmqe.web.tendrl.mainpage.dashboard.pages import Dashboard
+from usmqe.web.tendrl.mainpage.hosts.pages import HostsList
+from usmqe.web.tendrl.mainpage.file_shares.pages import FileSharesList
+from usmqe.web.tendrl.mainpage.pools.pages import PoolsList
+from usmqe.web.tendrl.mainpage.rbds.pages import RBDsList
+
+
+class ClusterMenu(WebstrPage):
+    """
+    cluster details menu
+    """
+    _model = m_cluster.ClusterMenuModel
+    _label = 'main page - clusters - cluster menu'
+    _required_elems = ['overview_link', 'hosts_link']
+
+    def open_overview(self, click_only=False):
+        """
+        open Overview tab
+
+        Parameters:
+            click_only (bool): if method should return Dashboard instance
+
+        Returns:
+            Dashboard instance if click_only is False
+        """
+        self._model.overview_link.click()
+        if not click_only:
```

Comment:
Ok, I just meant if it wouldn't be easier just to ignore output of this function and not to create a whole new parameter for it.